docs/workflow/optimize.md

```diff
@@ -1,104 +1,187 @@
-In addition this method helps us with debugging the optimization parameters. 
-
-#### Usage
-To run each of these optimization methods we need a proper set of parameters in a *.xml* file.
-The command to run the optimization with parameter files ``parameters.xml`` is as follows.
-```
-ShapeWorksRun parameters.xml
-``` 
-Here is the list of parameters and their descriptions.
-* inputs: Path to the directory containing the processed data in form of signed distance transform.
-* output_dir:  The directory where you need to save the output produced by the ShapeWorks optimization.
-* number of particles: (default:128) The desire number of particles to be placed in power of 2.
-* starting_particles: (_Only for multi-scale optimization_) The initial number of particles.
-* number_of_levels: (_Only for multi-scale optimization_) number of levels to run single scale optimization to reach desire number of particles.
-* iterations_per_split: The number of iterations in initialization step for each level of split. 
-* use_shape_statistics_in_init: (default: 1) uses the the statistics of shapes for next level initialization
-* save_init_splits: (default: 1) A boolean to save the particles foe each split in initialization stage. 
-* use_normals: A boolean variable for considering normal vector for each particle in optimization.
-* normal_weight: (if use_normal = True) The weights of normal vector for each axis (usually in scale of 10).
-* attribute_scales: The scale of each axis 
-* starting_regularization: (default:1000) Sets the starting regularization value (usually high value).
-* ending_regularization: (default:1e-4) Sets the ending regularization value (usually small value less than 1).
-* recompute_regularization_interval: Skip interval to compute the regularization value.
-* domains_per_shape: 
-* relative_weighting: (default: 1) Relative weight between the sampling and correspondence terms of the objective function
-* initial_relative_weighting: (default: 0.05) Starting relative weighting to make sure that initially the surfaces
- are covered uniformly (usually in order of ~0.1 or 0.01)
-* procrustes_scaling: (default: 0)  A boolean if the scaling in procrustes is to be enabled or not.
-* procrustes_interval: (default: 0) The interval between procrustes runs, 0 when procrustes is to be turned off.
+*This regularization can be considered as having a Gaussian ball in the shape space. Starting regularization pushes all samples to the mean and hides the underlying “unoptimized” covariance structure. Ending regularization should be small enough to reveal the optimized covariance structure.*
+![Starting and ending regularization](../img/workflow/regularization.png)
+
+## Optimizing Correspondences 
+
+You can use either ShapeWorksStudio or `shapeworks optimize <parameters.xml>` command to optimize your shape model. Both use a set of algorithmic parameters to control the optimization process.
+
+![Optimization Parameters](../img/workflow/params.png)
```

Comment:
the right half of the image: it says XML but looks like JSON. Maybe just set the header to "Optimization Parameters"

Reply:
fixed.